### ISSUE TITLE
UDP has maximum packet length of 8192 bytes only - Should be 65535

### ISSUE THREAD

**Issue by ryanhagen** [traefik/traefik#8547]
### Welcome!

- [X] Yes, I've searched similar issues on [GitHub](https://github.com/traefik/traefik/issues) and didn't find any.
- [X] Yes, I've searched similar issues on the [Traefik community forum](https://community.containo.us) and didn't find any.

### What did you do?

This may be a feature request. When I use a UDP router to route to a UDP service in the file provider, and send a UDP request larger than 8192 bytes, traefik only reads the first 8192 bytes. Traefik is running in a docker container, which is using UDP routing to route to a UDP service all defined in the file provider. The UDP service is on a separate physical machine than where traefik docker runs.

I would like to treafik to support a higher maximum packet length, say the maximum of the ipv4 spec 65535.

### What did you see instead?

I'm supplying a wireshark pcap as an exported csv, and redacting the source and dest IPs with the names of the services/clients. You can see that 8195 is sent to the traefik proxy, yet traefik only forwards 8192.

I've also verified the linux os has the correct udp_mem settings as I would expect.
```
cat /proc/sys/net/ipv4/udp_mem
udp_mem 765048  1020067 1530096
```
and core/rmem_max
```
cat /proc/sys/net/core/rmem_max
212992
```

```csv
"No.","Time","Source","Destination","Protocol","Length","Info"
"1","0.000000","CLIENT","PUBLIC_IP","IPv4","1518","Fragmented IP protocol (proto=UDP 17, off=0, ID=b14f) [Reassembled in #6]"
"2","0.000028","CLIENT","PUBLIC_IP","IPv4","1518","Fragmented IP protocol (proto=UDP 17, off=1480, ID=b14f) [Reassembled in #6]"
"3","0.000033","CLIENT","PUBLIC_IP","IPv4","1518","Fragmented IP protocol (proto=UDP 17, off=2960, ID=b14f) [Reassembled in #6]"
"4","0.000037","CLIENT","PUBLIC_IP","IPv4","1518","Fragmented IP protocol (proto=UDP 17, off=4440, ID=b14f) [Reassembled in #6]"
"5","0.000042","CLIENT","PUBLIC_IP","IPv4","1518","Fragmented IP protocol (proto=UDP 17, off=5920, ID=b14f) [Reassembled in #6]"
**"6","0.000046","CLIENT","PUBLIC_IP","UDP","841","57235  >  30000 Len=8195"**
"7","0.001159","TREAFIK_PROXY","UDP_SERVICE","IPv4","1518","Fragmented IP protocol (proto=UDP 17, off=0, ID=7b25) [Reassembled in #12]"
"8","0.001174","TREAFIK_PROXY","UDP_SERVICE","IPv4","1518","Fragmented IP protocol (proto=UDP 17, off=1480, ID=7b25) [Reassembled in #12]"
"9","0.001181","TREAFIK_PROXY","UDP_SERVICE","IPv4","1518","Fragmented IP protocol (proto=UDP 17, off=2960, ID=7b25) [Reassembled in #12]"
"10","0.001192","TREAFIK_PROXY","UDP_SERVICE","IPv4","1518","Fragmented IP protocol (proto=UDP 17, off=4440, ID=7b25) [Reassembled in #12]"
"11","0.001204","TREAFIK_PROXY","UDP_SERVICE","IPv4","1518","Fragmented IP protocol (proto=UDP 17, off=5920, ID=7b25) [Reassembled in #12]"
**"12","0.001211","TREAFIK_PROXY","UDP_SERVICE","UDP","838","59378  >  30000 Len=8192"**
"13","0.002360","UDP_SERVICE","TREAFIK_PROXY","IPv4","1454","Fragmented IP protocol (proto=UDP 17, off=0, ID=3ee6) [Reassembled in #18]"
"14","0.002376","UDP_SERVICE","TREAFIK_PROXY","IPv4","1454","Fragmented IP protocol (proto=UDP 17, off=1416, ID=3ee6) [Reassembled in #18]"
"15","0.002381","UDP_SERVICE","TREAFIK_PROXY","IPv4","1454","Fragmented IP protocol (proto=UDP 17, off=2832, ID=3ee6) [Reassembled in #18]"
"16","0.002392","UDP_SERVICE","TREAFIK_PROXY","IPv4","1454","Fragmented IP protocol (proto=UDP 17, off=4248, ID=3ee6) [Reassembled in #18]"
"17","0.002404","UDP_SERVICE","TREAFIK_PROXY","IPv4","1454","Fragmented IP protocol (proto=UDP 17, off=5664, ID=3ee6) [Reassembled in #18]"
"18","0.002413","UDP_SERVICE","TREAFIK_PROXY","UDP","1158","30000  >  59378 Len=8192"
"19","0.003039","TREAFIK_PROXY","CLIENT","IPv4","1518","Fragmented IP protocol (proto=UDP 17, off=0, ID=2d6b)"
"20","0.003052","TREAFIK_PROXY","CLIENT","IPv4","1518","Fragmented IP protocol (proto=UDP 17, off=1480, ID=2d6b)"
"21","0.003061","TREAFIK_PROXY","CLIENT","IPv4","1518","Fragmented IP protocol (proto=UDP 17, off=2960, ID=2d6b)"
"22","0.003072","TREAFIK_PROXY","CLIENT","IPv4","1518","Fragmented IP protocol (proto=UDP 17, off=4440, ID=2d6b)"
"23","0.003085","TREAFIK_PROXY","CLIENT","IPv4","1518","Fragmented IP protocol (proto=UDP 17, off=5920, ID=2d6b)"
"24","0.000053","TREAFIK_PROXY","CLIENT","UDP","838","30000  >  57235 Len=8192"
```

### What version of Traefik are you using?

Version:      2.5.3
Codename:     livarot
Go version:   go1.17.1
Built:        2021-09-20T15:43:56Z
OS/Arch:      linux/amd64

### What is your environment & configuration?

echoserviceudp is the udp router and service name in question.

Treafik.yml
```yaml
pilot:
    token: <token>
log:
  level: INFO
api:
  insecure: true
  dashboard: true
  debug: true
entryPoints:
  http:
    address: ":80"
    http:
      redirections:
        entryPoint:
          to: https
          scheme: https
  https:
    address: ":443"
  traefik:
    address: ":8080"
  zenohtcp:
    address: ":7447"
  zenohudp:
    address: ":7448/udp"
  echoservicetcp:
    address: ":30000"
  echoserviceudp:
    address: ":30000/udp"
serversTransport:
  insecureSkipVerify: true
providers:
  docker:
    endpoint: "unix:///var/run/docker.sock"
    exposedByDefault: false
  file:
    watch: true
    filename: config.yml
certificatesResolvers:
  myresolver:
    acme:
      email: <email>
      storage: letsencrypt/acme.json
      dnsChallenge:
        provider: cloudflare
        resolvers:
          - "1.1.1.1:53"
          - "1.0.0.1:53"
```
config.yml
```yaml
http:
  routers:
    work-kube:
      entryPoints:
        - "https"
      rule: "Host(`https://<dns>`)"
      service: work-kube
      tls:
        certResolver: myresolver
  services:
    work-kube:
      loadBalancer:
        servers:
          - url: "https://<ip>:16443"
        passHostHeader: true
udp:
  routers:
    echoserviceudp:
      entryPoints:
        - "echoserviceudp"
      service: "echoserviceudp"
  services:
    echoserviceudp:
      loadBalancer:
        servers:
          - address: "<ip>:30000"
tcp:
  routers:
    echoservicetcp:
      entryPoints:
        - "echoservicetcp"
      rule: "HostSNI(`*`)"
      service: "echoservicetcp"
  services:
    echoservicetcp:
      loadBalancer:
        servers:
          - address: "<ip>:30000"
```
Add more configuration information here.

**Comment by ryanhagen** [comment]
Please let me know if I should open this as a feature request.

**Comment by kevinpollet** [comment]
Closed by #8560
